4530692/.htaccess

```diff
@@ -0,0 +1,13 @@
+# Rewrite engine setup
+RewriteEngine on
+
+# Item within library in Web UI
+RewriteRule ^(.*) https://www.zotero.org/groups/4530692/usgs_ni_43-101_reports/items/$1 [R=302,L]
```

Comment:
This probably needs to go at the bottom.  I think it will match everything first before those cond rules are checked.

Reply:
Got it. Thanks. I made that change in the latest commit.